libs/application/templates/parental-leave/src/forms/ParentalLeaveForm.ts

```diff
@@ -403,11 +405,64 @@ export const ParentalLeaveForm: Form = buildForm({
           id: 'employer',
           title: parentalLeaveFormMessages.employer.subSection,
           children: [
-            buildCustomField({
-              component: 'SelfEmployed',
-              id: 'employer.isSelfEmployed',
-              title: parentalLeaveFormMessages.selfEmployed.title,
-              description: parentalLeaveFormMessages.selfEmployed.description,
+            buildMultiField({
+              id: 'employer.isSelfEmployed.benefits',
+              title: '',
+              children: [
+                buildCustomField({
+                  component: 'SelfEmployed',
+                  id: 'employer.isSelfEmployed',
+                  title: parentalLeaveFormMessages.selfEmployed.title,
+                  description:
+                    parentalLeaveFormMessages.selfEmployed.description,
+                }),
+                buildRadioField({
+                  id: 'isRecivingUnemploymentBenefits',
+                  title:
+                    parentalLeaveFormMessages.employer
+                      .isRecivingUnemploymentBenefitsTitle,
+                  description: '',
+                  width: 'half',
+                  options: [
+                    {
+                      label: parentalLeaveFormMessages.shared.yesOptionLabel,
+                      value: YES,
+                    },
+                    {
+                      label: parentalLeaveFormMessages.shared.noOptionLabel,
+                      value: NO,
+                    },
+                  ],
+                  condition: (answers) =>
+                    (answers as {
+                      employer: {
+                        isSelfEmployed: string
+                      }
+                    })?.employer?.isSelfEmployed === NO,
+                }),
+                buildSelectField({
+                  id: 'unemploymentBenefits',
+                  title: 'Hvaðan ertu að þyggja bætur?',
+                  options: [
+                    {
+                      label: unemploymentBenefits.vinnumálastofnun,
+                      value: unemploymentBenefits.vinnumálastofnun,
+                    },
+                    {
+                      label: unemploymentBenefits.stéttarfélagi,
+                      value: unemploymentBenefits.stéttarfélagi,
+                    },
+                    {
+                      label: unemploymentBenefits.sjúkratryggingarÍslands,
+                      value: unemploymentBenefits.sjúkratryggingarÍslands,
+                    },
+                  ],
```

Comment:
I think there are more than just these 3 which applicant could receive money from. Add 1 more option call 'Others'? ( Need clarification from VMST )

Reply:
These are the three we have on the jira board, guess we need to ask VMST